ducktape/tests/loader.py

```diff
@@ -230,34 +230,20 @@ def _parse_discovery_symbol(self, discovery_symbol, base_dir=None):
             "path/to/test_file.py" -> ("path/to/test_file.py", "", "")
             "path/to/test_file.py::ClassName.method" -> ("path/to/test_file.py", "ClassName", "method")
         """
+        def divide_by_symbol(ds, symbol):
```

Comment:
python's str.split() method already has this functionality - see `maxsplit` param:
https://docs.python.org/3/library/stdtypes.html#str.split

Reply:
cool!, I'll go ahead and use it.